src/platforms/java/common/async.mdx

```diff
@@ -13,7 +13,7 @@ Sentry's SDK for Java stores the scope and the context in a thread-local variabl
 
 To propagate Sentry scope to Spring MVC methods returning a `Callable` or a `StreamingResponseBody`, a `ThreadPoolTaskExecutor` decorated with `SentryTaskDecorator` must be set on the `AsyncSupportConfigurer` through an implementation of `WebMvcConfigurer`:
 
-```java
+```java {tabTitle:Java (Spring 5)}
```

Comment:
All these apply to Spring as well as Spring Boot. I tried putting `(Spring 5 / Spring Boot 2)` and `(Spring 6 / Spring Boot 3)` but that looks kinda bad.